api/app/Modules/Accounts/Events/Saves/SavedTrackRemoved.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Modules\Accounts\Events\Saves;
+
+use App\Modules\Library\Events\UserAction;
```

Comment:
Sounds like we need to move the `UserAction` class to the `App\Modules\Core\Events` namespace.

Reply:
Sure, will move this in a future PR